AltSwiftUI/AltSwiftUI/Source/Views/Controls/DatePicker.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  DatePicker.swift
+//  AltSwiftUI
+//
+//  Created by Chan, Chengwei on 2020/09/14.
+//  Copyright © 2020 Rakuten Travel. All rights reserved.
+//
+
+import UIKit
+
+/// A view that displays a wheel-style date picker that allows selecting
+/// a date in the range.
+public struct DatePicker: View {
+    public var viewStore = ViewValues()
+
+    var selection: Binding<Date>?
+    var minimumDate = Date()
+    var maximumDate: Date?
+    var components: Components
+    
+    public enum DatePickerComponent {
+        case date, hourAndMinute
```

Comment:
Please add documentation to each enum case

Reply:
updated